src/lib/components/submit/SubmitReview.svelte

```diff
@@ -44,39 +47,26 @@
 </script>
 
 {#if nonNullish(neuronId) && nonNullish(content)}
-	<form on:submit|preventDefault={onSubmit}>
-		<h1 class="font-bold capitalize text-4xl md:text-6xl mb-12">
-			Make sure everything looks good before submitting!
-		</h1>
-
-		<p class="leading-relaxed mb-8">
-			Review your proposal for Neuron ID: <Copy value={`${neuronId}`} text="Neuron ID copied." />.
-		</p>
-
-		<div in:fade>
-			<Container>
-				<aside slot="title">The proposal title</aside>
-				<article class="p-2.5">{metadata?.title ?? ''}</article>
-			</Container>
-
-			<HtmlMarkdown {content} />
+	<h1 class="mb-12 text-4xl font-bold capitalize md:text-6xl">
+		Make sure everything looks good before submitting!
+	</h1>
 
-			<Container>
-				<aside slot="title">An URL pointing to the forum</aside>
-				<article class="p-2.5">{metadata?.url ?? ''}</article>
-			</Container>
+	<p class="mb-8 leading-relaxed">
+		Review your proposal for Neuron ID: <Copy value={`${neuronId}`} text="Neuron ID copied." />.
+	</p>
 
-			<Container>
-				<aside slot="title">Your motion text</aside>
-				<article class="p-2.5">{metadata?.motionText ?? ''}</article>
-			</Container>
-		</div>
+	{#if $store?.metadata?.proposalAction === 'AddOrRemoveNodeProvider'}
+		<SubmitReviewAddNodeProvider metadata={$store?.metadata} />
+	{:else}
+		<form on:submit|preventDefault={onSubmit}>
```

Comment:
The `fade` effect has been removed, I am guessing it was there to smoothness the appearance. Can you please preserve it and also apply it for node provider?